core/metrics/src/service.rs

```diff
@@ -38,3 +40,25 @@ impl MetricsService {
 pub async fn expose_metrics(metrics: web::Data<Arc<Mutex<Metrics>>>) -> impl Responder {
     metrics.lock().await.export()
 }
+
+const PUSH_HOST_URL: &str = "http://127.0.0.1:9091/";
```

Comment:
Better to have it configurable

Reply:
env var + defgault to `http://metrics.golem.network:9091/`